assets/js/romo/indicator_text_input.js

```diff
@@ -68,20 +68,21 @@ RomoIndicatorTextInput.prototype._bindElem = function() {
   this.indicatorElem = $();
   var indicatorClass = this.elem.data('romo-indicator-text-input-indicator') || this.defaultIndicatorClass;
   if (indicatorClass !== undefined && indicatorClass !== 'none') {
-    this.indicatorElem = $('<i class="romo-indicator-text-input-indicator '+indicatorClass+'"></i>');
-    this.indicatorElem.romoIndicator();
+    this.indicatorElem = $('<div class="romo-indicator-text-input-indicator"><div><i class="'+indicatorClass+'"></i></div></div>');
```

Comment:
@kellyredding - Did you mean to include these changes? I don't think they were in the commit message.